nevergrad/optimization/recorded_recommendations.csv

```diff
@@ -122,6 +122,10 @@ MutDE,1.1400386808,0.3380024444,0.4755144618,2.6390460807,0.6911075733,1.1112355
 NGO,0.0,-0.3451057176,-0.1327329683,1.9291307781,,,,,,,,,,,,
 NGOpt,1.012515477,-0.9138805701,-1.029555946,1.2098418178,,,,,,,,,,,,
 NGOpt10,1.012515477,-0.9138805701,-1.029555946,1.2098418178,,,,,,,,,,,,
+NGOptSingle16,0.0,0.0,0.0,0.0,,,,,,,,,,,,
+NGOptSingle25,0.0,0.0,0.0,0.0,,,,,,,,,,,,
+NGOptSingle9,0.0,0.0,0.0,0.0,,,,,,,,,,,,
```

Comment:
looks like they all start by evaluating 0 :s

Reply:
Yes. Which is not bad I guess.